main.js

```diff
@@ -0,0 +1,51 @@
+const tweeter = Tweeter()
+const renderer = Renderer()
+
+renderer.renderPosts(tweeter.getPosts())
+
+
+$('#post').on('click', function () {
+  const $inputTwit = $('input').val()
+  if ($inputTwit) {
+    renderer.renderPosts(tweeter.addPost($inputTwit))
+    $('input').val('') 
+  } else { 
+    $('#input').attr('placeholder', "WRITE SMTH!")
+    $('#input').addClass('emptyInput') 
+
+    $('#input').on('focus', function () { 
+      $('#input').removeClass('emptyInput') 
+      $('#input').attr('placeholder', "What's on your mind?")
+    })
+  } 
+}
+)
+
+$('#posts').on('click', '.deleteTweetBTN', function () {
+  const postId = $(this).closest('.post').data('id')
+  renderer.renderPosts(tweeter.removePost(postId))
```

Comment:
See the note I wrote in the removePost method, you should call the getPosts method here